### PR TITLE
Cirrus: Add vendor + tree status check

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -61,7 +61,9 @@ validate_task:
         image: "${SKOPEO_CI_CONTAINER_FQIN}"
         cpu: 4
         memory: 8
-    script: make validate-local
+    script: |
+        make validate-local
+        make vendor && hack/tree_status.sh
```
Comment (mtrmac) [inline .cirrus.yml:66]: Does this need `set -e`, or splitting into two `script` instructions? It’s not obvious to me from the CI log.

**Review by mtrmac — COMMENTED** [review]
Thanks!

File: .cirrus.yml (hunk 1 above)
Comment (cevich) [reply .cirrus.yml:66]: Nope, not needed.  They do a `set -eo pipefail` by default for bash.

Comment (mtrmac) [reply .cirrus.yml:66]: Perfect, LGTM. Thanks again!

**Review by mtrmac — APPROVED** [review]

**Pull request merged**